pkg/pprof/pprof.go

```diff
@@ -0,0 +1,65 @@
+package pprof
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"net/http/pprof"
+	"time"
+
+	"github.com/go-logr/logr"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+)
+
+type Handler struct {
+	log      logr.Logger
+	bindAddr string
+}
+
+var (
+	_ manager.Runnable               = &Handler{}
+	_ manager.LeaderElectionRunnable = &Handler{}
+)
+
+func NewHandler(log logr.Logger, bindAddr string) *Handler {
+	return &Handler{
+		log:      log,
+		bindAddr: bindAddr,
+	}
+}
+
+func (h *Handler) NeedLeaderElection() bool {
+	return false
+}
+
+func (h *Handler) Start(ctx context.Context) error {
+	mux := http.NewServeMux()
+	mux.HandleFunc("/debug/pprof/", pprof.Index)
+	mux.HandleFunc("/debug/pprof/cmdline", pprof.Cmdline)
+	mux.HandleFunc("/debug/pprof/profile", pprof.Profile)
+	mux.HandleFunc("/debug/pprof/symbol", pprof.Symbol)
+	mux.HandleFunc("/debug/pprof/trace", pprof.Trace)
+
+	server := &http.Server{Addr: h.bindAddr, Handler: mux}
```

Comment:
I don't use `cybozu-go/well` to keep the log format consistent with the reconciler.
For this usage, the default http server is enough, in my opinion.

Reply:
I agree. 👍